assets/company-reports/actions.ts

```diff
@@ -1,6 +1,7 @@
 import {errorHandler, getItemFromArray, getDateInputDate, notify, gettext} from '../utils';
 import server from '../server';
 import {get, cloneDeep} from 'lodash';
+import {type ICompanyReportsData} from './types';
```

Comment:
we usually don't use import type specifically, is there any benefit?

Reply:
I don't think there isn't much benefit. Personally I prefer explicit than implicit but in this case it was because of the complain from TS I explained here https://github.com/superdesk/newsroom-core/pull/930#discussion_r1627162388.

Reply:
After applying the suggested changes here https://github.com/superdesk/newsroom-core/pull/930#discussion_r1627132852 the import is still required, otherwise TS will complain about undefined type.